src/main/kotlin/com/github/nowtilous/projectcolor/ui/ChooseColorAction.kt

```diff
@@ -1,26 +1,39 @@
 package com.github.nowtilous.projectcolor.ui
 
+import com.github.nowtilous.projectcolor.COLOR_SETTING_PATH
 import com.github.nowtilous.projectcolor.setTitleBarColor
+import com.intellij.ide.util.PropertiesComponent
 import com.intellij.openapi.actionSystem.AnAction
 import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.actionSystem.PlatformDataKeys
 import com.intellij.openapi.project.Project
+import com.intellij.ui.JBColor
+import java.awt.Color
+import java.awt.event.ActionListener
 import javax.swing.JColorChooser
 
-
 class ChooseColorAction : AnAction() {
     override fun actionPerformed(e: AnActionEvent) {
-
-        val colorChooser = JColorChooser()
         val project = e.dataContext.getData(PlatformDataKeys.PROJECT) as Project
+        val currentColor = PropertiesComponent.getInstance(project).getInt(COLOR_SETTING_PATH, getThemeBackgroundColor(e))
+
+        val colorChooser = JColorChooser(Color(currentColor))
```

Comment:
It's important to set the initial color of  _JColorChooser_ because that's the color that is going to be picked when the user clicks the "Reset" button. If you create the chooser using the default constructor - `JColorChooser()` -, the initial color is going to be white.